data/pid_data.py

```diff
@@ -784,7 +784,7 @@
        'notes': 'Enable / Disable the DHCP client.',
        'value': 0x800f},
 
-      #MAC_ADDRESS
+      # MAC_ADDRESS
       {'get_request': {'items': []},
        'get_response': {'items': [{'type': 'group',
                                    'name': 'languages',
```

Comment:
I don't know where this info came from initially, but I assume this is a typo @nomis52 and should be mac or something for the name?

Reply:
And in the set_request below?